### PR TITLE
Passive Buzzer Implementation

### PR DESCRIPTION
**Pull request by adelgadoj3**
Implemented the passive buzzer helper. Need to continue testing to obtain audio output from buzzer. Currently there is no audio output.

### CODE REVIEW

**Comment by joefol** [comment]
Make sure you are merging into develop and not main.

**Comment by adelgadoj3** [comment]
> Make sure you are merging into develop and not main.

Thanks for the catch Joe.

File: pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/PassiveBuzzerHelper.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
+package com.opensourcewithslu.outputdevices;
+
+import com.pi4j.io.pwm.Pwm;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+public class PassiveBuzzerHelper {
```
Comment (SwindleA) [inline pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/PassiveBuzzerHelper.java:10]: A class description is needed. javadoc

File: pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/PassiveBuzzerHelper.java [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+        this.passiveBuzzer = passiveBuzzer;
+
+    }
+
+    /**
+     * Initializes passive buzzer to a 50% duty cycle and frequency of 440Hz
+     */
+    //tag::method[]
+    public void passiveBuzzerOn(int passBuzzDC, int passiveBuzzerFreq){
```
Comment (SwindleA) [inline pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/PassiveBuzzerHelper.java:30]: parameter descriptions are needed. javadoc

**Review by SwindleA — CHANGES_REQUESTED** [review]
Looks good. a couple small javadoc issues. Builds correctly. I did not test it.

**Review by SwindleA — APPROVED** [review]
looks good

File: pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/PassiveBuzzerHelper.java [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+/**
+ * The PassiveBuzzerHelper class contains methods that pertain to the control of the passive buzzer.
+ */
+public class PassiveBuzzerHelper {
+
+    private static final Logger log = LoggerFactory.getLogger(PassiveBuzzerHelper.class);
+
+    private final Pwm passiveBuzzer;
+
+    protected int passiveBuzzerFreq = 440;
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/PassiveBuzzerHelper.java:20]: Variables which are not private requires Javadoc comments

File: pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/PassiveBuzzerHelper.java [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+           }
+       } catch (FileNotFoundException e) {
+           log.error("Frequencies file not found: " + frequenciesFile.getAbsolutePath());
+       }
+    }
+
+    /**
+     * Functionality test emits a 1 - second buzz to ensure functionality
+     */
+    public void functionalityTest(){
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/PassiveBuzzerHelper.java:114]: Test cannot be used as function names, change it

**Review by ruthvikm — CHANGES_REQUESTED** [review]
Need to change few function names where 'Test' is used, because in future we are going to write test cases for all the components. These function names leads to discrepancies.

**Review by ruthvikm — APPROVED** [review]

**Pull request merged**